src/Monolog/Processor/IntrospectionProcessor.php

```diff
@@ -55,6 +55,14 @@ public function __invoke(array $record)
             return $record;
         }
 
+        /*
+        * http://php.net/manual/en/function.debug-backtrace.php
+        * As of 5.3.6, DEBUG_BACKTRACE_IGNORE_ARGS option was added.
+        * Any version less than 5.3.6 must use the DEBUG_BACKTRACE_IGNORE_ARGS constant value '2'.
+        */
+        if (!defined('DEBUG_BACKTRACE_IGNORE_ARGS')) {
+            define('DEBUG_BACKTRACE_IGNORE_ARGS', 2);   
```

Comment:
This bug was introduced...
https://github.com/Seldaek/monolog/commit/cf2faeb3b73144666bb0e819b72b5bae977a9978